From 0df581d3b141b20b457e299c233e62157d333609 Mon Sep 17 00:00:00 2001
From: Ulrich Hecht <uli@basicengine.org>
Date: Thu, 21 Mar 2024 15:05:41 +0100
Subject: [PATCH] HACK: arm64: dts: allwinner: h616: disable GPU opp

No driver is implementing them on this SoC, causing panfrost probe to fail.
---
 arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
index 3d8826f47..925f75d67 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
@@ -1039,8 +1039,8 @@ gpu: gpu@0x01800000 {
 			clocks = <&ccu CLK_GPU0>, <&ccu CLK_BUS_GPU>;
 			clock-names = "core", "bus";
 			resets = <&ccu RST_BUS_GPU>;
-			operating-points-v2 = <&gpu_opp_table>;
-			#cooling-cells = <2>;
+			//operating-points-v2 = <&gpu_opp_table>;
+			//#cooling-cells = <2>;
 			status = "disabled";
 		};
 
@@ -1214,6 +1214,7 @@ map0 {
 			};
 		};
 
+#if 0
 		gpu-thermal {
 			polling-delay-passive = <0>;
 			polling-delay = <0>;
@@ -1240,6 +1241,7 @@ map0 {
 				};
 			};
 		};
+#endif
 
 		ve-thermal {
 			polling-delay-passive = <0>;
-- 
2.39.2

